packages/paste-core/components/page-header/stories/index.stories.tsx

```diff
@@ -0,0 +1,287 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable import/no-extraneous-dependencies */
+import { Anchor } from "@twilio-paste/anchor";
+import { Avatar } from "@twilio-paste/avatar";
+import { Badge } from "@twilio-paste/badge";
+import { Box } from "@twilio-paste/box";
+import { Breadcrumb, BreadcrumbItem } from "@twilio-paste/breadcrumb";
+import { Button } from "@twilio-paste/button";
+import { ButtonGroup } from "@twilio-paste/button-group";
+import { CustomizationProvider } from "@twilio-paste/customization";
+import { MoreIcon } from "@twilio-paste/icons/esm/MoreIcon";
+import { InPageNavigation, InPageNavigationItem } from "@twilio-paste/in-page-navigation";
+import { Paragraph } from "@twilio-paste/paragraph";
+import {
+  ProgressStepComplete,
+  ProgressStepCurrent,
+  ProgressStepIncomplete,
+  ProgressStepSeparator,
+  ProgressSteps,
+} from "@twilio-paste/progress-steps";
+import { StatusBadge } from "@twilio-paste/status";
+import { useTheme } from "@twilio-paste/theme";
+import * as React from "react";
+
+import {
+  PageHeader,
+  PageHeaderActions,
+  PageHeaderContext,
+  PageHeaderDetails,
+  PageHeaderHeading,
+  PageHeaderInPageNavigation,
+  PageHeaderKeyword,
+  PageHeaderMeta,
+  PageHeaderParagraph,
+  PageHeaderPrefix,
+} from "../src";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "Components/Page Header",
+  component: PageHeader,
+};
+
+export const Default = (): React.ReactElement => {
+  return (
+    <PageHeader size="default">
+      <PageHeaderContext>
+        <Breadcrumb>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+        </Breadcrumb>
+      </PageHeaderContext>
+      <PageHeaderDetails>
+        <PageHeaderKeyword>Wizard title</PageHeaderKeyword>
+        <PageHeaderPrefix>
+          <Avatar size="sizeIcon90" name="Avatar Name" />
+        </PageHeaderPrefix>
+        <PageHeaderHeading>Page title</PageHeaderHeading>
+        <PageHeaderActions>
+          <ButtonGroup>
+            <Button variant="secondary">Action</Button>
+            <Button variant="secondary">Action</Button>
+          </ButtonGroup>
+        </PageHeaderActions>
+        <PageHeaderMeta>
+          <Badge variant="success" as="span">
+            Badge
+          </Badge>
+          Meta
+          <Anchor href="#">Meta</Anchor>
+        </PageHeaderMeta>
+        <PageHeaderParagraph>
+          <Paragraph marginBottom="space0">Paragraph text</Paragraph>
```

Comment:
suggestion: I notice is some instances you pass a `Paragraph` component to `PageHeaderParagraph`, in others you don't. It feels almost redundant to pass a paragraph to a component called paragraph. Should it just be a string, always? Does `PageHeaderParagraph` have all the styles `Paragraph` sets? should `Paragraph` be an internal child to `PageHeaderParagraph`, wrapping `{children}`?

Reply:
good catch, it does feel redundant. fixed it to match PageHeaderHeading, so the PageHeaderParagraph is a box wrapper around the actual Paragraph component